pipelines/HyperSuprimeCam/ApPipeWithFakes.yaml

```diff
@@ -19,8 +19,20 @@ imports:
       - detectAndMeasure
       - diaPipe
       - transformDiaSrcCat
+parameters:
+  coaddName: goodSeeing
+  fakesType: 'fakes_'
```

Comment:
Sorry I missed this before, but why this override relative to `_ingredients/ApPipeWithFakes`? It will cause unexpected behavior if we ever need to change defaults.